src/main/java/com/internet/shop/servise/impl/OrderServiceImpl.java

```diff
@@ -1,7 +1,56 @@
 package com.internet.shop.servise.impl;
 
+import com.internet.shop.dao.interfaces.OrderDao;
+import com.internet.shop.dao.interfaces.ShoppingCartDao;
+import com.internet.shop.db.Storage;
+import com.internet.shop.lib.Inject;
 import com.internet.shop.lib.Service;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.servise.interfaces.OrderService;
+import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
-public class OrderServiceImpl {
+public class OrderServiceImpl implements OrderService {
+
+    @Inject
+    private OrderDao orderDao;
+
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        List<Product> products = List.copyOf(shoppingCart.getProducts());
+        Long userId = shoppingCart.getUserId();
+        Order order = new Order(userId);
+        order.setProducts(products);
+        Storage.addOrder(order);
```

Comment:
- we can't use Storage at the service layer
- let's call method clear() on the shoppingCart